packages/bff/src/services/catalogService.ts

```diff
@@ -294,5 +294,43 @@ export function catalogServiceBuilder(
         id: updatedEservice.id,
       };
     },
+    createEService: async (
+      eServiceSeed: bffApi.EServiceSeed,
+      { headers }: WithLogger<BffAppContext>
+    ): Promise<bffApi.CreatedResource> => {
+      const { id } = await catalogProcessClient.createEService(
+        toCatalogCreateEServiceSeed(eServiceSeed),
+        {
+          headers,
+        }
+      );
+      return { id };
```

Comment:
In Scala this returns a `CreatedEServiceDescriptor`, not a `CreatedResource` - it's an object containing both the `id` and the `descriptorId`. Weird that TS does not complain... maybe you did not pull the latest change that modified this? I see it was done a few weeks ago: https://github.com/pagopa/interop-be-backend-for-frontend/commit/283b909fec85f4ab241b6751ed93f375cd5184d1

See the call to `toApiWithDescriptorId` here 👇 
https://github.com/pagopa/interop-be-backend-for-frontend/blob/b37f24cc27d7690cc9ece8dd215a16437de2f0cf/src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala#L93-L94

Reply:
[e917276](https://github.com/pagopa/interop-be-monorepo/pull/885/commits/e917276d9d0c88a13950a6991d57e0f250c85af3)